cfpb/consumerfinance.gov
# P4C: Conditional content updates

Hide PELL and SEOG for grad students and update conditional display of salary data.

## Changes

- Apply undergrad class to options unavailable to grad students
- Use class instead of data attribute to display general salary data when program not selected 


## How to test this PR

1. Check out branch 
2. Navigate to grad path tool and verify that PELL and SEOG content do not appear when graduate student is selected
3. Verify that program-specific salary data appears when program is selected, and general salary data appears when program is not selected


## Screenshots



## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
